tools/README.md

```diff
@@ -0,0 +1,21 @@
+# Archive Node Developer Tools
+
+<hr>
+
+Some tools to make working with Mina Archive Nodes and their SQL dumps provided by O(1) a little easier
+
+<hr>
+
+## Download Archive Dump
+
+downloads the latest archive dump for your selected network
+
+* build: `nix-build`
+* run: `./result/bin/download-archive-dump --help`
```

Comment:
Nice that there is a built-in '--help'!

Reply:
In might be nice to know from where the archive dump comes without looking at the code, IMO.